packages/components/organisms/f-cookie-banner/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-cookie-banner",
   "description": "Fozzie Cookie Banner – Cookie Banner",
-  "version": "2.2.1",
+  "version": "2.2.2",
```

Comment:
I think we should bump the minor version as a new prop has been introduced as well as a fix?

Reply:
Yes, I was thinking the same. Will update. Thanks